atitto/galereya/interier.php

```diff
@@ -365,58 +365,7 @@
 </div>
 
 
-<footer>
-<div class="footcomp">
-<div class="row">
-
-<div class="col-md-12 col-xs-12 col-lg-12 col-sm-12" style="text-align:center; margin-left:-9%">
-
-<p><font color='black'><strong> <span style="font-size: 50%; color: black"></span>
-<img src="http://1.bp.blogspot.com/-4njwXch0jpc/Ut6EnB8TCiI/AAAAAAAAEN8/ChxzjoPZImw/s50/Communication-gmail-icon.png" width='3%' height='3%' class="imageslogos">
-<span> atittosk@gmail.com </span><span style="color: black"></span>
-<img src="/telefon4.png" width='3%' height='3%'>
-<span> +421 908-52-90-11 </span>
-
- <img src="https://lh6.ggpht.com/mp86vbELnqLi2FzvhiKdPX31_oiTRLNyeK8x4IIrbF5eD1D5RdnVwjQP0hwMNR_JdA=w50" width='3%' height='3%' class="imageslogos">
- <span> +421 908-52-90-11 </span>
- <p><img src="https://lh3.googleusercontent.com/8WsuI2NpvYJljZshzCgLXKRLnFLzz3q4I3JGw3L0vfintPGEiTChOXDIrnoXFFfme58=w50"  width='3%' height='3%' class="imageslogos" >
- +380 675-62-41-11 </span>
- <a href="https://www.facebook.com/Atitto-Dreven%C3%A9-domy-s-modernou-izol%C3%A1ciou-441945522814419/?ref=aymt_homepage_panel">
- <img src="/facebook.png"  width='10%' height='3%' class="imageslogos" > 
- </a></p></span></strong>
-<p>Jánošíkovo nábrežie, 4478, 031 01 Liptovský Mikuláš, Slovenský Republika</p>
-
-</font>
-</div>
-</div>
-</div>
-<div class="foottel">
-<div class="row">
-
-<div class="col-md-12 col-xs-12 col-lg-12 col-sm-12" style="text-align:center; margin-left:-9%">
-
-<p><font color='black'><strong> <span style="font-size: 50%; color: black"></span>
-<img src="http://1.bp.blogspot.com/-4njwXch0jpc/Ut6EnB8TCiI/AAAAAAAAEN8/ChxzjoPZImw/s50/Communication-gmail-icon.png" width='10%' height='10%' class="imageslogos">
-<span> atittosk@gmail.com </span><span style="color: black"></span>
-
-<p><img src="/telefon4.png" width='10%' height='10%'>
-<span> +421 908-52-90-11 </span>
-
- <p><img src="https://lh6.ggpht.com/mp86vbELnqLi2FzvhiKdPX31_oiTRLNyeK8x4IIrbF5eD1D5RdnVwjQP0hwMNR_JdA=w50" width='10%' height='10%' class="imageslogos">
- <span> +421 908-52-90-11 </span>
- <p><img src="https://lh3.googleusercontent.com/8WsuI2NpvYJljZshzCgLXKRLnFLzz3q4I3JGw3L0vfintPGEiTChOXDIrnoXFFfme58=w50"  width='10%' height='10%' class="imageslogos" >
- +380 675-62-41-11 </span>
- <a href="https://www.facebook.com/Atitto-Dreven%C3%A9-domy-s-modernou-izol%C3%A1ciou-441945522814419/?ref=aymt_homepage_panel">
- <p><img src="/facebook.png"  width='30%'  class="imageslogos" > 
- </a></p></span></strong>
-<p>Jánošíkovo nábrežie, 4478, 031 01 Liptovský Mikuláš, Slovenský Republika</p>
-
-</font>
-</div>
-</div>
-</footer>
-
-</body>
+<?php include('../footer.php'); ?>
```

Comment:
in our project right now footer is the part of body. so you've removed closing tag for body. that will not work)